packages/reactivity/src/computed.ts

```diff
@@ -68,10 +68,51 @@ export class ComputedRefImpl<T> {
   }
 }
 
+/**
+ * Creates an immutable reactive ref object.
+ *
+ * The `getter` function's return value determines the ref's value.
+ *
+ * @example
+ * ```js
+ * const count = ref(1)
+ * const plusOne = computed(() => count.value + 1)
+ *
+ * console.log(plusOne.value) // 2
+ * plusOne.value++ // error
+ * ```
+ *
+ * @param getter Function that produces the next value.
```

Comment:
To follow TSdocs, we should also apply this to all params in all functions

```suggestion
 * @param getter - Function that produces the next value.
```

Reply:
👍